test/tarball/verifySemVer.spec.js

```diff
@@ -0,0 +1,61 @@
+// Require Node.js Dependencies
+import { describe, test } from "node:test";
+import assert from "node:assert";
+import { getSemVerWarning } from "../../src/utils/index.js";
+
+// CONSTANTS
+const KVersionsToWarn = new Set(["0.0.0", "0.0", "0"]);
+
+const [firstVersion, secondVersion, lastVersion] = KVersionsToWarn.values();
```

Comment:
```suggestion
const [fullVersion, noPatchVersion, noMinorVersion] = KVersionsToWarn.values();
```
Something like this ?
I'm not sure theses constants are really necessary tbh